frontend/src/GridCell.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import Grid from "@mui/joy/Grid";
+
+interface Props {
+    children?: React.ReactNode;
```

Comment:
I think I'm missing why `children` is optional? What is a GridCell with no children?

Reply:
Shoot my bad, I had a `?` there at first because I was using it to specify an empty `GridCell` (for cells in the grid which should remain empty) but then I ended up preferring passing in an empty `<div />` to `children` for empty cells, so the `?` should have been removed.

The latest commit removes `GridCell` since having it be a separate component became a little pointless, but if that commit is reverted since it's not the best solution, will remove the `?`!